src/styles/learn.scss

```diff
@@ -52,11 +52,9 @@
   padding: calc(var(--space-12) / 2) 0 calc(var(--space-12) / 2) var(--space-08);
   position: relative;
   text-decoration: none;
-
```

Comment:
@arun-kushwaha04 I still see a bunch of unrelated to the task changes - like these lines removal. Can you please leave only related to side nav changes?

Reply:
@rodion-arr These line of code are already present in my branch.
https://github.com/arun-kushwaha04/nodejs.dev/blob/cebece442080b3b8bcfb1e52b7ca3cb597c3ea9b/src/styles/learn.scss#L52-L54

Reply:
I mean, beside your changes related to side nav - you've removed a bunch of empty lines that I'd like to keep on it's places.

Reply:
Oh, I am really sorry that misunderstanding. I have added spaces just like It was earlier.